java/ql/lib/change-notes/2024-06-14-reverse-dns-separate-threat-model-kind.md

```diff
@@ -0,0 +1,4 @@
+---
+category: majorAnalysis
+---
+* We previously considered reverse DNS resolutions (IP address -> domain name) as sources of untrusted data, since compromised/malicious DNS servers could potentially return malicious responses to arbitrary requests. We have now removed this source from the default set of untrusted sources and made a new threat model kind for them, called "reverse-dns".
```

Comment:
Might make sense to include an example of how to use this new threat model.

Reply:
I imagine it depends on how you are running CodeQL (code-scanning, CLI, extension), so it might be best to link to some docs for this. (Do we put links in change notes? And do we have docs on this?)